src/app/containers/Metadata/index.jsx

```diff
@@ -15,6 +17,37 @@ const allTags = tags => {
   return aboutTags.concat(mentionTags);
 };
 
+const checkType = types => {
+  if (types.length > 1 && types.includes('Thing')) {
+    return 'Thing';
+  }
+  return types[0];
+};
+
+const checkSameAs = uris =>
+  uris.filter(uri => uri.includes('http://dbpedia.org'));
+
+const aboutTagsContent = aboutTags => {
```

Comment:
It would be better to move this `aboutTags` logic into its own JS file within this container, to have more unit tests around the input and output of JSON. `/src/app/containers/Metadata/linkedDataAbout.js`

For example, testing that the only Type values that should be added are 'Person', 'Place', 'Event' and 'Organization'
- If a tag has multiple types provided by the data feed it should set the '@type' value as 'Thing'. 
- If a tag has no types provided it should set the '@type' value as 'Thing'.